src/features/account/components/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './security-reports/SecurityReportsTabPanel';
+export * from './reviews/ReviewsTabPanel';
+export * from './trusted-circle/TrustedCircleTabPanel';
+export * from './snaps/MySnapsTabPanel';
```

Comment:
These should have a `index.ts` in each folder which exports the component.

Reply:
Do they need to be in a separate folder in the first place? It seems like each folder has just a single component.

Reply:
it will have more components in the up coming story